blueflood-http/src/main/java/com/rackspacecloud/blueflood/http/DefaultHandler.java

```diff
@@ -46,18 +46,18 @@ public static void sendResponse(ChannelHandlerContext channel, HttpRequest reque
             Iterator<String> itr = headers.keySet().iterator();
             while(itr.hasNext()){
                 String headerKey = itr.next();
-                response.setHeader(headerKey,headers.get(headerKey));
+                response.setHeader(headerKey, headers.get(headerKey));
             }
         }
-        final Timer.Context sendResponseTimerContext = sendResponseTimer.time();
 
+        final Timer.Context sendResponseTimerContext = sendResponseTimer.time();
         try {
             if (messageBody != null && !messageBody.isEmpty()) {
                 response.setContent(ChannelBuffers.copiedBuffer(messageBody, Constants.DEFAULT_CHARSET));
             }
 
-            Tracker.getInstance().trackResponse(request, response);
             HttpResponder.respond(channel, request, response);
+            Tracker.getInstance().trackResponse(request, response);
```

Comment:
I would like to be able to track the response headers too, so have to move it after the respond call where the headers are set.